usr.bin/sockstat/sockstat.c

```diff
@@ -1510,7 +1492,27 @@ display(void)
 		err(1, "malloc()");
 		return;
 	}
-	calculate_column_widths(&cw);
+
+	cw = (struct col_widths) {
+		.user = opt_w ? strlen("USER") : 8,
+		.command = 10,
+		.pid = opt_w ? strlen("PID") : 5,
+		.fd = opt_w ? strlen("FD") : 3,
+		.proto = opt_w ? strlen("PROTO") : 6,
+		.local_addr = opt_w ? strlen("LOCAL ADDRESS") : 21,
+		.foreign_addr = opt_w ? strlen("FOREIGN ADDRESS") : 21,
+		.pcb_kva = 18,
+		.fib = opt_w ? strlen("FIB") : 6,
+		.splice_address = opt_w ? strlen("SPLICE ADDRESS") : 21,
+		.inp_gencnt = opt_w ? strlen("ID") : 8,
+		.encaps = opt_w ? strlen("ENCAPS") : 6,
+		.path_state = opt_w ? strlen("PATH STATE") : 12,
+		.conn_state = opt_w ? strlen("CONN STATE") : 12,
+		.stack = opt_w ? strlen("STACK") : TCP_FUNCTION_NAME_LEN_MAX,
+		.cc = opt_w ? strlen("CC") : TCP_CA_NAME_MAX,
```

Comment:
`TCP_CA_NAME_MAX` is also too wide.  The values for these two widths in the previous revision, 5 and 2, were better.